docs/src/content/docs/fr/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Encarts
+description: Apprenez à utiliser les encarts dans Starlight pour afficher des informations secondaires à côté du contenu principal d'une page.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Pour afficher des informations secondaires à côté du contenu principal d'une page, utilisez le composant `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="aperçu">
+Incluez des informations supplémentaires non essentielles dans un `<Aside>`.
+</Aside>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Aside } from '@astrojs/starlight/components';
+```
+
+## Utilisation
+
+Affichez un encart à l'aide du composant `<Aside>`.
+
+Un `<Aside>` peut avoir un attribut facultatif [`type`](#type), qui contrôle la couleur, l'icône et le titre par défaut de l'encart.
```

Comment:
```suggestion
Le composant `<Aside>` peut avoir un attribut facultatif [`type`](#type), qui contrôle la couleur, l'icône et le titre par défaut de l'encart.
```

Same as before, I think this would only work if the component was named `<Encart>`.